Makefile

```diff
@@ -62,6 +62,13 @@ MULTICALL_${CONFIG_IFQUERY}_OBJ += ${IFQUERY_SRC:.c=.o}
 CMDS_${CONFIG_IFQUERY} += ifquery
 CPPFLAGS_${CONFIG_IFQUERY} += -DCONFIG_IFQUERY
 
+# enable ifctrstat applet (+16 KB)
```

Comment:
is the +16 KB actually accurate here?

Reply:
Yes.  GCC 8 with default CFLAGS (`-Os -ggdb3`).  `CONFIG_IFCTRSTAT=N` vs `Y` changes the size of `ifupdown`:

```
awilcox on catbookpro [pts/0 Fri 14 11:28] ifupdown-ng: ls -al ifupdown
-rwxr-xr-x 1 awilcox awilcox 214040 Aug 14 11:28 ifupdown
awilcox on catbookpro [pts/0 Fri 14 11:28] ifupdown-ng: make CONFIG_IFCTRSTAT=Y
...
awilcox on catbookpro [pts/0 Fri 14 11:28] ifupdown-ng: ls -al ifupdown        
-rwxr-xr-x 1 awilcox awilcox 230888 Aug 14 11:28 ifupdown
```

Reply:
I've been measuring the stripped version delta.  Most likely this is +4KB at most.